packages/opentelemetry-core/test/context/HttpTraceContext.test.ts

```diff
@@ -173,8 +203,6 @@ describe('HttpTraceContext', () => {
 
       const testCases: Record<string, string> = {
         invalidParts_tooShort: '00-ffffffffffffffffffffffffffffffff',
-        invalidParts_tooLong:
```

Comment:
This test shouldn't be needed anymore because future versions may have extra fields, and the new regex doesn't currently capture any of the new fields anyway.